src/caret_analyze/runtime/application.py

```diff
@@ -449,13 +449,13 @@ def get_node_path(
         ----------
         node_name : str
             node name to get.
-        subscribe_topic_name : str | None
+        subscribe_topic_name : str
             topic name which the node subscribes.
-        publish_topic_name : str | None
+        publish_topic_name : str
             topic name which the node publishes.
-        subscription_construction_order : int | None
+        subscription_construction_order : int
             A construction order of subscription.
-        publisher_construction_order : int | None
+        publisher_construction_order : int
             A construction order of publisher.
```

Comment:
Changes to the source code should be made in a separate PR.

Reply:
Supported by the PR (https://github.com/tier4/caret_analyze/pull/490). Please review.